### ISSUE TITLE
Unable to cancel dropPane mode filepicker

### ISSUE THREAD

**Issue by kaypro4** [filestack/filestack-js#364]
Hello, I'm trying to cancel in progress uploads using the dropPane mode of filepicker. 

I'm using the .cancel() method on the picker instance and it is returning the onCancel method but when I look in the filestack content browser the files are fully uploaded.

Is there a way to both cancel the upload and remove the file from filestack as part of that cancel?

Relevant code in my project:

```
const client = filestack.init(FILESTACK_KEY,config)
                
const options = {
                    container: '#droppane',
                    displayMode: 'dropPane',
                    maxFiles: 1,
                    accept: this.accept,
                    lang: locale,
                    uploadInBackground: false,
                    dropPane: {
                        overlay: true,
                        onSuccess: (res) => {
                            this.fileName = res[0].filename;
                            this.item.fileHandle = res[0].handle;
                            this.missingFile = false;
                        }
                    },
                    onFileUploadStarted: (res) => {
                        this.uploading = true;
                    },
                    onUploadDone: (res) => {
                        this.uploading = false;
                    },
                    onCancel: (res) => {
                        console.log(res);
                        console.log('cancelled');
                    },
                };
                
                this.pickerInstance = client.picker(options);
                this.pickerInstance.open();
```

Then triggering this on mid upload cancel by user

```
this.pickerInstance.cancel();
```

Package version is 3.12.4

Thanks for any assistance!

**Comment by jeremyadavis** [comment]
I'd love some clarity around this as well. 

We're starting to dive into some of our production issues relating to our use of FileStack and noticed this behavior as well. Photos that are 'canceled' mid-upload via closing the overlay still continue to upload and are stored in our s3 bucket.

My understand is that closing the overlay would stop the upload and this [old comment](https://github.com/filestack/filestack-js/issues/18#issuecomment-318105083) from FileStack suggests that as well.

We're on the 3.16.0 and I've tried several versions back to see if it was a regression, but it behaves the same in the ones I've tried.

Even the official [live examples](https://github.com/filestack/filestack-js#live-examples-jsfiddle) seem to continue to upload after 'cancelling' so I'm a bit perplexed as to how to proceed to prevent this behavior.
